code/modules/industrial_lift/tram_controls.dm

```diff
@@ -110,8 +118,19 @@
 		return FALSE
 	tram_part.tram_travel(to_where)
 	say("The tram has been called to [to_where.name].")
+	update_appearance()
 	return TRUE
 
+/obj/machinery/computer/tram_controls/proc/update_tram_display(obj/effect/landmark/tram/from_where, travelling)
+	SIGNAL_HANDLER
+	var/datum/lift_master/tram/tram_part = tram_ref?.resolve()
+	if(!travelling)
```

Comment:
Either invert this (so that you don't have "if-not-else") or make it a ternary (also inverted)

Reply:
Done.